### PR TITLE
File Utilities class tests

### PR DESCRIPTION
**Pull request by gmwasi**

### CODE REVIEW

**Comment by keymane** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- opensrp-app/src/test/java/org/smartregister/util/FileUtilitiesTest.java  2
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/OpenSRP/opensrp-client-core/pullRequest?prid=5510166&bid=18059684)

File: opensrp-app/src/test/java/org/smartregister/util/FileUtilitiesTest.java [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+
+    @Test
+    public void initIndexFile_validFile_addsEmptyraces() throws Exception {
+        String testData = "string to write";
+        when(Environment.getExternalStorageDirectory()).thenReturn(existentDirectory);
+        PowerMockito.whenNew(BufferedWriter.class).withAnyArguments().thenReturn(mockWriter);
+        FileUtilities fileUtils = new FileUtilities();
+        try
+        {
+            fileUtils.write(FILE_NAME, testData);
```
Comment (githengi) [inline opensrp-app/src/test/java/org/smartregister/util/FileUtilitiesTest.java:51]: There is no check that this is written

**Comment by keymane** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- opensrp-app/src/test/java/org/smartregister/util/FileUtilitiesTest.java  2
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/OpenSRP/opensrp-client-core/pullRequest?prid=5510166&bid=18059684)

**Comment by keymane** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- opensrp-app/src/test/java/org/smartregister/util/FileUtilitiesTest.java  2
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/OpenSRP/opensrp-client-core/pullRequest?prid=5510166&bid=18059684)

**Comment by keymane** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- opensrp-app/src/test/java/org/smartregister/util/FileUtilitiesTest.java  2
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/OpenSRP/opensrp-client-core/pullRequest?prid=5510166&bid=18059684)

**Comment by keymane** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- opensrp-app/src/test/java/org/smartregister/util/FileUtilitiesTest.java  2
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/OpenSRP/opensrp-client-core/pullRequest?prid=5510166&bid=18059684)

**Comment by keymane** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- opensrp-app/src/test/java/org/smartregister/util/FileUtilitiesTest.java  2
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/OpenSRP/opensrp-client-core/pullRequest?prid=5510166&bid=18059684)

**Comment by keymane** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- opensrp-app/src/test/java/org/smartregister/util/FileUtilitiesTest.java  2
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/OpenSRP/opensrp-client-core/pullRequest?prid=5510166&bid=18059684)

**Comment by keymane** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- opensrp-app/src/test/java/org/smartregister/util/FileUtilitiesTest.java  2
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/OpenSRP/opensrp-client-core/pullRequest?prid=5510166&bid=18059684)

**Comment by keymane** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- opensrp-app/src/test/java/org/smartregister/util/FileUtilitiesTest.java  2
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/OpenSRP/opensrp-client-core/pullRequest?prid=5510166&bid=18059684)

**Pull request merged**